GitHub Notifications.xcodeproj/project.pbxproj

```diff
@@ -416,9 +416,10 @@
 				COMBINE_HIDPI_IMAGES = YES;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"GitHub Notifications/Preview Content\"";
-				DEVELOPMENT_TEAM = 68JNFYH5JF;
+				DEVELOPMENT_TEAM = S7245DPLBK;
```

Comment:
This screams: "gitignore me", might need to switch (if not already the case) to build a "swift package" and get rid of this: https://stackoverflow.com/questions/8026429/should-i-git-ignore-xcodeproject-project-pbxproj-file